portfolio/src/main/webapp/script.js

```diff
@@ -35,13 +42,13 @@ function topFunction() {
 // https://www.quora.com/How-can-I-make-a-comment-box-in-HTML
 // This code gets comment data from the /data backend path and updates the #display-comments element
 // with comment data.
-fetch('/data').then(response => response.json()).then((data) => {
+fetch('/data').then(response => response.json()).then((comments) => {
```

Comment:
Good renaming and the iteration over the comments here is clearer

Reply:
im learning! haha